src/main/sensors/gyro.c

```diff
@@ -823,7 +989,15 @@ int16_t gyroGetTemperature(void)
 
 int16_t gyroRateDps(int axis)
 {
+#ifdef USE_DUAL_GYRO
+    if (gyroToUse == GYRO_CONFIG_USE_GYRO_2) {
+        return lrintf(gyro.gyroADCf[axis] / gyroSensor2.gyroDev.scale);
+    } else {
+        return lrintf(gyro.gyroADCf[axis] / gyroSensor1.gyroDev.scale);
```

Comment:
Most code like this in BF I've seen is like:
```cpp
#ifdef USE_DUAL_GYRO
if (gyroToUse == GYRO_CONFIG_USE_GYRO_2) {
  return /* ... */ ;
} else
#endif
{
  return /* ... */;
}
```

to avoid duplication